src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/Processors/LedgerTransactionMarkers/ManifestProcessor.cs

```diff
@@ -108,128 +108,169 @@ public void OnTransactionScan(CoreModel.CommittedTransaction transaction, long s
                 _referencedEntities.MarkSeenAddress(address);
             }
 
-            _manifestExtractedAddresses.Add(stateVersion, extractedAddresses);
+            if (transaction.Receipt.Status == CoreModel.TransactionStatus.Succeeded)
+            {
+                _manifestExtractedAddresses.Add(stateVersion, extractedAddresses);
```

Comment:
Out of interest, why do we only include this, and not e.g. `_referencedEntities` above? What is the `_referencedEntities` check used for?

Reply:
As discussed, we can probably move this if statement to cover `_referencedEntities.MarkSeenAddress(address);` as well, but we need to check for any edge cases.

Reply:
moved.  We still need to extract manfiest to do classification as we do that for failed tx as well.